cmd/kots/cli/velero.go

```diff
@@ -81,11 +81,14 @@ func VeleroEnsurePermissionsCmd() *cobra.Command {
 				return errors.Wrap(err, "failed to get clientset")
 			}
 
-			if err := snapshot.EnsureVeleroPermissions(cmd.Context(), clientset, veleroNamespace, kotsadmNamespace); err != nil {
-				return err
+			skipRbac := v.GetBool("skip-rbac")
```

Comment:
can we have this consistent with the other commands? which i believe is called `ensure-rbac`

Reply:
Would they need then specificy `--ensure-rbac false` and the default would be true?

Reply:
`--ensure-rbac=false`, and yes.

Reply:
Feels a little clunky, how about a compromise.  We use `skip-rbac-check` already for other commands, how do you feel about me changing `skip-rbac` to `skip-rbac-check`?